### DISCUSSION TITLE
WT32-ETH01 PHY Reset Missing - In your experience is needed ?

### DISCUSSION THREAD

**Discussion by FulvioSpelta** [arendst/Tasmota#20745] [Q&A]
Hi all I've found this note: [https://github.com/dok-net/wt32-eth01-uart/tree/main](https://github.com/dok-net/wt32-eth01-uart/tree/main)
that tells: 

> The WT32-ETH01, at least until version 1.4, doesn't reset the PHY. This can result in a hanging PHY on occasion, which cannot be remedied by a reset of the ESP32, but only by a full power cycle

Does anyone have experienced some kind of problems related to this on long term use of the WT32-ETH01 module with tasmota ?
Thanks for comments.

**Comment by sfromis** [comment]
I've not experienced trouble needing power cycling with the 1.2 board. My testing has not been extensive.
